src/ua/hillel/Dog.java

```diff
@@ -0,0 +1,29 @@
+public class Dog extends Animals{
+     int max_run =500;
+     int max_swim =10;
+     int add_dog;
```

Comment:
Would be better to rename `add_dog` to `dogCounter`. Make this variable private static.